js/ui/select_box.js

```diff
@@ -399,10 +399,11 @@ var SelectBox = DropDownList.inherit({
 
         dataSourceIsLoaded.done((function() {
             var selectedIndex = this._getSelectedIndex(),
+                hasPages = this._dataSource.pageSize(),
```

Comment:
```suggestion
                isPageLoad = this._dataSource.paginate(),
```

Reply:
It doesn't work. isLastPage = !paginate(), so this sugg doesn't change the behavior